### PR TITLE
`[PRD]` | `[Issue #2545]` add more socials

### LINKED ISSUE #2545: add more socials
https://www.notion.so/hulking-cosmos-3e8/Q4-2024-QA-Requirements-13e2ee9c5bc280a88458e7a935d17488?pvs=4#13e2ee9c5bc280229892c0d678980b3e

### PR DESCRIPTION
**Pull request by Da-Colon**
Closes #2545 

- removes warcast icon from sidebar
- add social links to footer
- adjust spaces in footer
- `BugFix` ExternalLink green pill's hover state

localhost_8888_(Desktop)

### CODE REVIEW

File: src/components/ui/links/ExternalLink.tsx [hunk 1]
```diff
@@ -105,35 +105,31 @@ export default function ExternalLink({
   };
 
   return (
-    <Flex>
```
Comment (Da-Colon) [inline src/components/ui/links/ExternalLink.tsx:108]: Don't need to wrap in a `Flex`

File: src/components/ui/links/ExternalLink.tsx [hunk 2]
```diff
@@ -105,35 +105,31 @@ export default function ExternalLink({
...
-        target="_blank"
-        rel="noreferrer"
-        textDecoration="none"
-        ref={internalRef}
-        {...rest}
+        alignItems="center"
       >
-        <Flex
-          gap="0.25rem"
-          mx="-0.75rem"
```
Comment (Da-Colon) [inline src/components/ui/links/ExternalLink.tsx:130]: Not really sure why this was here? The hover state's padding was being squeezed by this.

File: src/constants/url.ts [hunk 3]
```diff
@@ -1,5 +1,8 @@
 export const URL_FAQ = 'https://docs.decentdao.org/app/learn-more/faq';
-export const URL_CHAT = 'https://warpcast.com/~/channel/decentdao';
+export const URL_WARPCAST = 'https://warpcast.com/~/channel/decentdao';
 export const URL_DOCS = 'https://docs.decentdao.org/app';
 export const URL_DOCS_GOV_TYPES = 'https://docs.decentdao.org/app/user-guide/types';
 export const URL_DECENT = 'https://www.decentdao.org/';
+export const URL_TWITTER = 'https://twitter.com/'; // @todo - Add Twitter handle;
+export const URL_TELEGRAM = 'https://t.me/'; // @todo - Add Telegram handle;
+export const URL_DISCORD = 'https://discord.gg/'; // @todo - Add Discord handle;
```
Comment (Da-Colon) [inline src/constants/url.ts:8]: @nicolaus-sherrill Can I get the official links to use?

**Review by mudrila — CHANGES_REQUESTED** [review]
Looks good despite missing actual links

**Comment by nicolaus-sherrill** [comment]
@Da-Colon 
Twitter: https://twitter.com/decentdao
Discord: https://discord.gg/Zh2emKspVF

Like Kirill said, TG is closed so we need an admin to provide a link.

**Comment by parkermccurley** [comment]
Approved - thanks @Da-Colon

**Comment by parkermccurley** [comment]
Approved

**Review by mudrila — APPROVED** [review]

**Review by DarksightKellar — APPROVED** [review]

**Pull request merged**